pkg/distributors/test/MerkleRedeem.test.ts

```diff
@@ -277,8 +277,56 @@ describe('MerkleRedeem', () => {
       expect(result).to.eql(expectedResult); // "claim status should be accurate"
     });
 
+    interface Claim {
+      week: BigNumber;
+      balance: BigNumber;
+      merkleProof: BytesLike[];
+    }
+
+    describe('with a callback', () => {
+      let callbackContract: Contract;
+      let claims: Claim[];
+
+      sharedBeforeEach('set up mock callback', async () => {
+        callbackContract = await deploy('MockRewardCallback');
+
+        const proof1: BytesLike[] = merkleTree1.getHexProof(elements1[0]);
+        const proof2: BytesLike[] = merkleTree2.getHexProof(elements2[0]);
+
+        claims = [
+          { week: bn(1), balance: claimBalance1, merkleProof: proof1 },
+          { week: bn(2), balance: claimBalance2, merkleProof: proof2 },
+        ];
+      });
+
+      it('allows a user to claim the reward to a callback contract', async () => {
```

Comment:
Let's add a test that checks that callback has been granted the tokens as internal balance

Reply:
I believe that's what this test does!